### PR TITLE
Safari 16.5 is released

### PR DESCRIPTION
**Pull request by Elchi3**
https://webkit.org/blog/14154/webkit-features-in-safari-16-5/

I don't know how to set the correct engine_version. Anyone has pointers?

### CODE REVIEW

**Comment by queengooborg** [comment]
> I don't know how to set the correct engine_version. Anyone has pointers?

I'll update my laptop later today and get the engine version for you!

**Comment by Elchi3** [comment]
> > I don't know how to set the correct engine_version. Anyone has pointers?
> 
> I'll update my laptop later today and get the engine version for you!

Nudge :) Would be good to get this in.

File: browsers/safari.json [hunk 1]
```diff
@@ -251,13 +251,14 @@
...
+          "status": "retired",
           "engine": "WebKit",
           "engine_version": "615.1.26"
         },
         "16.5": {
+          "release_date": "2023-05-18",
           "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-16_5-release-notes",
-          "status": "beta",
+          "status": "current",
           "engine": "WebKit"
```
Comment (queengooborg) [inline browsers/safari.json:262]:
```suggestion
          "engine": "WebKit",
          "engine_version": "615.2.9"
```

File: browsers/safari_ios.json [hunk 2]
```diff
@@ -223,13 +223,14 @@
...
+          "status": "retired",
           "engine": "WebKit",
           "engine_version": "615.1.26"
         },
         "16.5": {
+          "release_date": "2023-05-18",
           "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-16_5-release-notes",
-          "status": "beta",
+          "status": "current",
           "engine": "WebKit"
```
Comment (queengooborg) [inline browsers/safari_ios.json:234]:
```suggestion
          "engine": "WebKit",
          "engine_version": "615.2.9"
```

**Review by queengooborg — COMMENTED** [review]
Here we go!  (I hadn't updated my laptop yet since I was working, but I have done so now!)

**Review by queengooborg — APPROVED** [review]

**Pull request merged**